contributors/join-us/recruiting.md

```diff
@@ -1,28 +1,44 @@
 
-## Hiring in new U.S. States
+## Hiring across the globe
 
-Each time we hire in a new U.S. state we may be subject to new regulations, taxes, and reporting obligations and we need time to prepare for compliant operations in new states ahead of doing business in state, which begins when we employ a new hire.
+### We currently support hiring in the following countries/regions:
 
-For U.S. states not on the above list the recruiting team should provide at least 3 weeks' notice ahead of a potential offer signing date, indicating that a new state would be added.
+* Canada, Finland, Georgia, Germany, India, Philippines, South Africa, Turkey, United Kingdom, United States
 
-To add a new U.S. state please DM @tim.quock to provide the Finance team with advance notice, and indicate if a lead time of more than 3 weeks may be needed.
+We are constantly working towards adding more countries/regions to this list, but it takes a lot of time and resources. This list will be updated as we are able to hire in additional countries/regions. 
 
-## We are not yet able to hire in the following regions/countries
+Due to local laws, there may be restrictions to which types of roles we can hire in each country/region.
 
-We're continuing to find ways to hire in more regions/countries, here are some that are more difficult than others:
+### Hiring in the United States
 
-Brazil, Iran, Russian Federation, China, Syria, Lebanon, North Korea
+We currently have staff in the following U.S. states:
+
+* AZ CA CO FL GA IN KS MA MD MI MO NJ NV NY OK SC TN UT VA WA WI
+
+Preparing to hire an employee in a new state takes time, and we may be subject to new regulations, taxes, and reporting obligations. If a candidate is progressing in the interview process and would be living and working from a U.S. state not listed above, the Recruiter will DM @tim.quock with the state and the title of the position the candidate may fill at least 3 weeks prior to an anticipated offer. 
+
+### Relocating
+
+Staff agreements are conditioned upon the region of residence noted in a staff member's contract. Location change requests should be discussed with HR and your MLT member. Staff must receive written approval from the Company prior to a region relocation.
+
+## Lever
+
+Lever is our Applicant Tracking System (ATS). It is where our Recruiters, Recruiting Coordinators, and Hiring Managers organize our open roles, track a candidate’s progress throughout the recruiting process, collect feedback, and record data. It is important that all candidates, interviews, and feedback are recorded accurately in Lever because all of our data and insights are pulled from Lever. Without accurate data in Lever, we aren’t able to iterate on the process to deliver a better hiring experience for candidates, hiring managers, and the business overall. 
```

Comment:
```suggestion
Lever is our Applicant Tracking System (ATS). It is where our Recruiters, Recruiting Coordinators, and Hiring Managers organize open roles, track a candidate’s progress throughout the recruiting process, collect feedback, and record data. It is important that all candidates, interviews, and feedback are recorded accurately in Lever because all of our data and insights are pulled from Lever. Without accurate data in Lever, we aren’t able to iterate on the process to deliver a better hiring experience for candidates, hiring managers, and the business overall. 
```